src/envo/envo-edit.owl

```diff
@@ -39756,6 +39758,29 @@ AnnotationAssertion(Annotation(rdfs:comment "In German \"Kunststoff\" connotes t
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105101> "plastic"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105101> <http://purl.obolibrary.org/obo/ENVO_00010483>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105107> (marine anthropogenic litter)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105107> "Anthropogenic litter which is present in the marine or coastal environment."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105107> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105107> <https://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105107> <http://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105107> "2021-05-04T14:07:44Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105107> "海洋廢棄物"@zh)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105107> "envoPlastics")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105107> "marine anthropogenic litter"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105107> <http://purl.obolibrary.org/obo/ENVO_03500005>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105108> (littering)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.3390/environments7100073>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105108> "A waste disposal process during which waste is improperly disposed of into the environment."@en)
```

Comment:
Please add a comment on this class and the continuant litter class that "litter" typically refers to "small" objects (e.g. [here](https://www.dictionary.com/browse/litter)). Not sure what the threshold is (I don't think there is one, the semantics here are very loose), but I wouldn't consider a fridge "litter".

Reply:
@pbuttigieg when working on "anthropogenic litter" we had a comment stating that:

> very large objects that are discarded are typically considered "debris", but this term is loosely defined and there are no standard size classifications which can be used to distinguish debris from litter.

Should we use the same comment for the sake of consistency?